.github/workflows/release.yml

```diff
@@ -28,13 +28,18 @@ jobs:
       - name: Check out repo
         uses: actions/checkout@v4
 
-      - name: Install Java and Maven
+      - name: Set up JDK 8
         uses: actions/setup-java@v3
         with:
           java-version: 8
           distribution: temurin
           check-latest: false
 
+      - name: Set up Maven
+        uses: stCarolas/setup-maven@v4.5
+        with:
+          maven-version: 3.9.4
```

Comment:
How was this even working before?!

Reply:
`setup-java` sets up a default version of Maven as well!